src/fs-plus.coffee

```diff
@@ -60,6 +60,21 @@ fsPlus =
       return "#{fsPlus.getHomeDirectory()}#{relativePath.substring(1)}"
     return relativePath
 
+  # Public: Convert an absolute path to tilde path for linux and mac:
+  # /Users/username/dev => ~/dev
+  #
+  # pathToTildify - The {String} containing the full path.
+  #
+  # Returns a tildified path {String}.
+  tildify: (pathToTildify) ->
+    if process.platform is 'win32'
```

Comment:
👍 

I opted for the multiline version since I find it more readable but thats preference.